src/transactions/page.rs

```diff
@@ -10,6 +10,34 @@ use crate::common::search::*;
 use crate::common::spotlight::*;
 use crate::icons::*;
 
+#[component]
+pub fn TransactionTabbedPage() -> impl IntoView {
+    let tabs = vec![
+        NavEntry {
+            href: "/transactions".to_string(),
+            text: "Transactions".to_string(),
+            icon: NavIcon::Transactions,
+            ..Default::default()
+        },
+        NavEntry {
+            href: "/transactions/token-trxs".to_string(),
+            text: "Token Trnxs".to_string(),
+            icon: NavIcon::Tokens,
+            disabled: true,
+            ..Default::default()
+        },
+        NavEntry {
+            href: "/transactions/zk-trxs".to_string(),
+            text: "zkApp Trnxs".to_string(),
+            icon: NavIcon::ZKApps,
+            disabled: true,
+            ..Default::default()
+        },
+    ];
```

Comment:
This data looks similar to [src/header/navigation.rs](https://github.com/Granola-Team/mina-block-explorer/pull/275/files#diff-0e1780afc8d4e29558d37d1f542f0c8d495a966e65b6ef9da6bec80a4760d957). Is there a way to share that data so we have it in a single place?

Reply:
It is similar, but conceptually they are different. One set of `NavEntry` represents the dropdown menu, and another set represent the tabs on the page. It's not always true that drowdown menus will always be additionally represented as tabs on the page.